js/lib/ilib.js

```diff
@@ -794,6 +794,6 @@ ilib.isDynData = function() {
  * merging is too slow, so the already-merged data is cached as well after the first time it
  * is requested. In this way, we sacrifice the memory footprint for the sake of speed.
  */
-if (ilib._getPlatform() !== 'qt') ilib._cacheMerged = false;
+ilib._cacheMerged = false;
```

Comment:
Should be `ilib._cacheMerged = (ilib._getPlatform() === 'qt');`

Reply:
yeah..but If I add this line `ilib.js`. It makes lots of test case failures.